app/services/apiUtils.ts

```diff
@@ -1,4 +1,11 @@
 import { generateApiClient } from '@app/utils/apiUtils';
 const ituneApi = generateApiClient('itune');
 
-export const getItune = (param: any) => ituneApi.get(`/search?term=${param}`);
+type ApiParamProps = {
+  artistName: string;
+  pageNumber: number;
+  pageSize: number;
+};
+
+export const getItune = ({ artistName, pageNumber = 1, pageSize = 10 }: ApiParamProps) =>
+  ituneApi.get(`/search?term=${artistName}&offset=${pageNumber}&limit=${pageSize}`);
```

Comment:
It's better to use the params object that [axios](https://axios-http.com/docs/req_config) supports